libensemble/manager.py

```diff
@@ -534,6 +521,22 @@ def _final_receive_and_kill(self, persis_info):
 
     # --- Main loop
 
+    def _get_alloc_libE_info(self):
+        "Selected statistics useful for alloc_f"
+
+        libE_info = {'exit_criteria': self.exit_criteria,
+                     'elapsed_time': self.elapsed(),
+                     'manager_kill_canceled_sims': self.kill_canceled_sims,
```

Comment:
Are you going to be modifying this PR to show where each of these can be used?

Reply:
I am going to describe each of these fields in the docs and suggestions on how they could be used in an allocation function, but were you thinking of allocation function examples?

Reply:
I dont think we had a specific case for these, but in give_sim_work_first I think we were going to do Jeff's new cancel block after sim_max_given. 

We also talked about bringing the condition "if any(self.W['active'] == 0)" inside here - perhaps git could be made a boolean?

Reply:
Okay - it will get points_to_evaluate so may as well have a quick check at the top.